recipes/refet/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "refet" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "1b614ffbdaa44bcdb1869d8c2095583e6509a0cb6be952d643893a4f55dce220" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - refet
+
+about:
+  home: https://github.com/DRI-WSWUP/RefET
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.txt
```

Comment:
Consider adding the `license_file` to the source distribution in future releases instead of having to manually package it with the recipe.

Reply:
If you look in the tarball on pypi, the license file actually isn't included in the source distribution. I believe the proper way to handle this is to add a `MANIFEST.in` to the repo that includes the license file:

https://docs.python.org/2/distutils/sourcedist.html#specifying-the-files-to-distribute

It's fine for now. I'll merge and you can change it in the feedstock in a future release if you add it.